### PR TITLE
increase memory available to INSAR_GAMMA jobs in azdwr-hyp3 deployment

### PR DESCRIPTION
**Pull request by asjohnston-asf**
Change from INSAR_GAMMA.yml:
- increased memory to 63200
- increased omp-num-threads to 8

```
$ diff INSAR_GAMMA.yml INSAR_GAMMA_HIGH_MEMORY.yml 
75c75
<         - '4'
---
>         - '8'
99c99
<       memory: 31600
---
>       memory: 63200
```

### CODE REVIEW

File: job_spec/INSAR_GAMMA_HIGH_MEMORY.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+INSAR_GAMMA:
```
Comment (jtherrmann) [inline job_spec/INSAR_GAMMA_HIGH_MEMORY.yml:1]: Maybe? I don't actually know if this should change.
```suggestion
INSAR_GAMMA_HIGH_MEMORY:
```

Comment (jtherrmann) [inline job_spec/INSAR_GAMMA_HIGH_MEMORY.yml:1]: Maybe? I don't actually know if this should change.
```suggestion
INSAR_GAMMA_HIGH_MEMORY:
```

Comment (asjohnston-asf) [reply job_spec/INSAR_GAMMA_HIGH_MEMORY.yml:1]: The job_type you give to the API is driven by the top-level dictionary key, not the file name. This lets azdwr-hyp3 continue to use the same job name (`INSAR_GAMMA`) but get the higher memory reservation.

We do the same thing with AUTORIFT.yml and AUTORIFT_ITS_LIVE.yml, so jobs in both edc-prod and hyp3-autorift are named `AUTORIFT`, but are run with different configurations.

Comment (jtherrmann) [reply job_spec/INSAR_GAMMA_HIGH_MEMORY.yml:1]: Cool, thanks for clearing that up.

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**